src/net/http/headers.rs

```diff
@@ -0,0 +1,50 @@
+use std::collections::HashMap;
+
+#[derive(Default, Debug, Clone)]
+pub struct Headers {
+    headers: HashMap<String, String>,
+}
+
+impl Headers {
```

Comment:
This is a very simple encapsulation of a hashmap. I think reqwest has a different way that may or may not be better.. but for now this will do.